py-polars/polars/lazyframe/frame.py

```diff
@@ -140,7 +140,7 @@ class LazyFrame:
         Two-dimensional data in various forms; dict input must contain Sequences,
         Generators, or a `range`. Sequence may contain Series or other Sequences.
     schema : Sequence of str, (str,DataType) pairs, or a {str:DataType,} dict
-        The DataFrame schema may be declared in several ways:
+        The LazyFrame schema may be declared in several ways:
```

Comment:
Techincally, you are declaring a `DataFrame` schema when initialising a `LazyFrame` through this class, however I think this update reduces potential confusion in the reader.